### DISCUSSION TITLE
Middle school student

### DISCUSSION THREAD

**Discussion by Cyberwolfdev** [community/community#17695] [GitHub Education]
Is a middle school eligible for GitHub edu

**Comment by AveyBD** [comment]
No. 
As per GitHub Education requirements the students age should be minimum 13 years.

**Comment by Cyberwolfdev** [comment]
I am 13



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by v1ctorio** [comment]
You probably are, just fill out the form and wait. Keep in mind that you must pass the requirements of GitHub like being 13 years old. Also in the proof part, you can just attach a screenshot of your school portal.
